components/meter/README.md

```diff
@@ -139,6 +123,21 @@ Circle meters will show the progress as a full circle.
 	* `\{\%\}` in the string will be replaced with percentage value
 	* `{x/y}` in the string will be replaced with fraction with the proper language support
 
+## Light Foreground
```

Comment:
Looks like this would now be hidden. Is that what's desired?

Reply:
Think I put it one block too low, nice catch thanks!